themes/default/content/docs/guides/organizing-projects-stacks/_index.md

```diff
@@ -106,6 +106,28 @@ its associated Pulumi stack. Read more about
 
 Stacks have associated metadata in the form of name/value tags. You can assign custom tags to stacks (when logged into the [Pulumi Service backend]({{< relref "/docs/intro/concepts/state" >}})) to customize how stacks are listed in the [Pulumi Service](https://app.pulumi.com). For example, if you have many projects with separate stacks for production, staging, and testing environments, it may be useful to group stacks by environment instead of by project. To do this, you could assign a custom `environment` tag to each stack, assigning a value of `production` to each production stack, `staging` to each staging stack, etc. Then in the Pulumi Service, you'll be able to group stacks by `Tag: environment`. For more information, see [Stack tags]({{< relref "/docs/intro/concepts/stack#stack-tags" >}}).
 
-## Examples
+## Example
+
+In practice, it's common to have a combination of approaches. This is an example of an orgaizational setup that leverages several different approaches to provide the most functionality and flexibility possible.
+
+We start with a central base "infrastructure" project, which contains things that are common across multiple services (or perhaps even your entire organization!). This project can include resources like Azure Resource Groups or AWS VPCs.
+
+Within this project, we create stacks for each unique configuration (often times stacks are related to SDLC environments like dev, staging, and production). These stacks are often deployed independently of each other and are often deployed in different regions. To use a metaphor, our Pulumi program code defines the shape of our dial, and the configuration in the different stack configuration files (e.g., `Pulumi.dev.yaml`, `Pulumi.staging.yaml`, `Pulumi.prod.yaml`) defines the actual dial setting. These "dial settings" might include things like subscription IDs, regions, etc. that are specific to that environment.
+
+This project lays down our base infrastructure. One of the advantages to keeping this separate is there is likely a limited number of users we want to be able to deploy these things; not every indvidual team needs to be able to do this.
+
+Now that we have our base infrastructure, we can create a separate Pulumi project per application or service for each one's deployment and configuration that will include all the resources that the service needs, which are not provided by the base infrastructure project.
+
+Our example service is made up of an API and a database (RDS, CosmosDB, etc.). Our Pulumi program for the project defines the resources for the API and the database, and it can also deploy the actual code, as well.
+
+It's generally a good practice to keep our projects on the smaller side as this helps reduce the effect of a deployment (i.e., the micro-stacks model described earlier).
```

Comment:
I think we need to elaborate here. When I advise customers, I talk about two things:

- Rate of change
- Blast radius

It might be worth a separate sub page on these topics

Reply:
for the record, I removed the term "blast radius" per @nimbinatus suggestion as it is considered violent imagery; the "effect of deployment" is the replacement for this.

rate of change should be added! ty!!!

Reply:
added mention of change rate, etc